Web/Annotation Assignment/AnnotationAssignment2/src/Employee.java

```diff
@@ -0,0 +1,50 @@
+
+/**
+ * @class Employee
+ * @author Pulkit
+ * @since: 16 October,2015 
+ * This class defines members of the employee.
+ **/
+
+import lombok.Data;
+
+// Used lambok api for defining setters and getters.
+
+@Data
+public class Employee {
+	
+	String employeeName;
+	int employee_id;
+	String email_id;
+	String date_Of_Birth;
```

Comment:
follow proper naming conventions i.e remove underscores and keep symmetry for uppercase or lower case letters